app/models/api_docs_search.rb

```diff
@@ -1,25 +0,0 @@
-module ApiDocsSearch
```

Comment:
Let's keep this file. Our current APIs may not be using it, but there is a plan/desire to resurrect and refactor document collection searches via API.